wiki/en/en-Hardware-Setup.md

```diff
@@ -25,14 +25,14 @@ Setting up hardware usually burns down to 4 points, although each setup is diffe
 
 ## Windows: Audio interface connection - ASIO4ALL
 
-This is an example Windows client installation with audio device [Behringer U-CONTROL UCA202](https://www.amazon.com/Behringer-U-Phono-UFO202-Audiophile-Interface/dp/B002GHBYZ0).
+This is an example Windows client installation with the [Behringer U-CONTROL UCA202](https://www.amazon.com/Behringer-U-Phono-UFO202-Audiophile-Interface/dp/B002GHBYZ0) audio device.
 The following instructions might be similar with other audio devices.
```

Comment:
`audio device`. Which brings us back to the question if we should use interface/audio decice/sound card,... Not sure about the outcome.

Reply:
Me neither... that's why I left it as-is.

Reply:
This was my take https://github.com/jamulussoftware/jamuluswebsite/issues/168#issuecomment-743317373 (which would mean that this amend is fine)

Reply:
ok. Will resolve this